vector/src/main/java/im/vector/app/features/attachments/AttachmentsHelper.kt

```diff
@@ -214,7 +214,7 @@ class AttachmentsHelper(val context: Context, val callback: Callback) : Restorab
                         it.toContentAttachmentData()
                     }
             )
-        } else if (type.startsWith("application") || type.startsWith("file") || type.startsWith("*")) {
+        } else if (type.startsWith("application") || type.startsWith("file") || type.startsWith("text") || type.startsWith("*")) {
```

Comment:
do we want to explicitly handle all the possible cases? or should our else case allow handling any type as a file?

Reply:
For me i prefer to explicitly handle all the possible cases to keep the control of what we send. It's easy to add a new type if a user need it.